tests/phpunit/StreamWrapper/IncludeInterceptorTest.php

```diff
@@ -223,4 +224,20 @@ public function test_passthrough_dir_methods_pass(): void
 
         IncludeInterceptor::disable();
     }
+
+    public function test_it_re_enables_interceptor_after_file_not_found(): void
+    {
+        IncludeInterceptor::intercept(self::$files[1], self::$files[2]);
+        IncludeInterceptor::enable();
+
+        try {
+            fopen('file-does-not-exist.txt', 'r');
+        } catch (Warning $e) {
```

Comment:
Would `$this->expectException(Warning::class)` do the trick here as well?

(I think phpunit 8.4 introduced a `expectWarning`, but i'm not sure what phpunit version we run atm)